playbooks/roles/postgresql_aggregation/tasks/main.yml

```diff
@@ -6,18 +6,32 @@
     install_recommends: yes
     state: present
   with_items: postgresql_aggregation_packages
```

Comment:
@amirtds noticed this when trying to run this role. The above variable should be enclosed in double brackets. Otherwise it's evaluated as a string.